pretyping/coercionops.mli

```diff
@@ -31,9 +31,19 @@ val subst_cl_typ : env -> substitution -> cl_typ -> cl_typ
 (** Comparison of [cl_typ] *)
 val cl_typ_ord : cl_typ -> cl_typ -> int
 
+module ClTypSet : Set.S with type elt = cl_typ
+
 (** This is the type of infos for declared classes *)
 type cl_info_typ = {
-  cl_param : int }
+  (* The number of parameters of the coercion class. *)
+  cl_param : int;
+  (* The sets of coercion classes respectively reachable from and to the
+     coercion class. *)
+  cl_reachable_from : ClTypSet.t;
+  cl_reachable_to : ClTypSet.t;
+  (* The representative class of the strongly connected component. *)
+  cl_repr : cl_typ;
+}
```

Comment:
We don't need to expose all this stuff, just expose a function `class_nparams : cl_typ -> int` and remove `class_info`, `cl_info_typ` and `ClTypSet`